devito/propagator.py

```diff
@@ -342,11 +342,16 @@ def sympy_to_cgen(self, stencils):
             stencil = self.convert_equality_to_cgen(equality)
             stmts.append(stencil)
 
+        nest_decl = lambda dec, stmt: cgen.Assign(dec.__str__()[:-1], stmt.rvalue)
+
+        for idx, dec in enumerate(decl):
+            stmts[idx] = nest_decl(dec, stmts[idx])
```

Comment:
str(dec) will return `float temp;` Hence the last element is `;`. I wanna get rid of the `;`.
and then combine them together